website_sale_suggest_create_account/static/tests/tours/checkout.js

```diff
@@ -0,0 +1,45 @@
+odoo.define("wbesite_sale_suggest_create_account.shop_buy", function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+
+    tour.register(
+        "shop_buy_checkout_suggest_account_website",
```

Comment:
Please, complete this tour...

Reply:
Tour is complete. All the steps are followed.Let me know the steps which is missed from my end